src/app/leverage/page.tsx

```diff
@@ -1,9 +1,19 @@
+import { LeverageChart } from './components/leverage-chart'
+import { LeverageWidget } from './components/leverage-widget'
+import { Stats } from './components/stats'
 import { Title } from './components/title'
 
 export default function Page() {
   return (
-    <div className='mx-auto p-12'>
-      <Title />
+    <div className='mx-auto flex flex-col gap-6 p-12'>
```

Comment:
`mx-auto` doesn't seem to center horizontally here. my guess is it's because I have to set the width somewhere. can't really seem to find where tho. maybe you got an idea @0xonramp?

Reply:
@janndriessen yes I believe so, should work if you apply something like `max-w-7xl` to it